Description: Fix sqlalchemy.exceptions import.
 Use ‘exceptions’ as a normal attribute of the ‘sqlalchemy’ module, don't
 try to import it. (The ‘sqlalchemy.exceptions’ alias in sys.modules was
 removed in SQLAlchemy 0.7.)
Author: Jakub Wilk <jwilk@debian.org>
Bug-Debian: http://bugs.debian.org/638747
Forwarded: no
Last-Update: 2011-09-16

--- a/SPDataManager.py
+++ b/SPDataManager.py
@@ -38,7 +38,6 @@
 
 try:
     import sqlalchemy as sqla
-    import sqlalchemy.exceptions as sqlae
     import SQLTables
 except ImportError:
     module_logger.exception("No sqlalchemy package found")
@@ -82,7 +81,7 @@
                 sqltb = SQLTables.SqlTables(self.metadata)
                     # returns lookup table to get table objects by name and creates tables
                 self.tables = sqltb._create_tables(self.metadata, self.engine)
-            except (AttributeError, sqlae.SQLAlchemyError), info:
+            except (AttributeError, sqla.exceptions.SQLAlchemyError), info:
                 self.logger.exception("Failed to start the DBase, %s" % info)
                 raise MyError, info
             else:
